opc/scripts/setup/wizard.py

```
@@ -565,23 +604,30 @@ async def run_setup_wizard() -> None:
-    config = {"database": db_config, "api_keys": api_keys}
+    # Step 5: Generate .env
+    console.print("\n[bold]Step 5/13: Generating configuration...[/bold]")
+    config = {"database": db_config, "embeddings": embeddings, "api_keys": api_keys}
     env_path = Path.cwd() / ".env"
     generate_env_file(config, env_path)
     console.print(f"  [green]OK[/green] Generated {env_path}")
 
     # Step 5: Container stack (Sandbox Infrastructure)
     runtime = prereqs.get("container_runtime", "docker")
-    console.print(f"\n[bold]Step 5/12: Container Stack (Sandbox Infrastructure)[/bold]")
+    console.print(f"\n[bold]Step 6/13: Container Stack (Sandbox Infrastructure)[/bold]")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)